ocs_ci/ocs/elasticsearch.py

```diff
@@ -42,14 +42,22 @@ def elasticsearch_load(connection, target_path):
         log.error("There is No data to load into ES server")
         return False
     else:
+        if connection is None:
+            log.warning("There is no elasticsearch server to load data into")
+            return False
+        """
         log.info(f"Loading data from {target_path} to ES server at {connection}")
+        log.info(f"Creating new ES connection to {connection}")
         try:
-            log.info(f"Creating new ES connection to {connection}")
-            main_es = Elasticsearch([connection])
-            log.debug(f"Connection info : {main_es}")
-        except esexp.ConnectionError:
+            main_es = Elasticsearch([connection], verify_certs=True)
+        except esexp.ConnectionError as ex:
+            log.error(f"Can not connect to the main ES server on {connection} - ({ex})")
```

Comment:
Spelling fix - Cannot instead of Can not

Reply:
fixed it